### ISSUE TITLE
Rules Conflict / Nested interactive controls are not announced by screen readers

### ISSUE THREAD

**Issue by deleonio** [dequelabs/axe-core#3155]
I think there is a conflict beetween the rule and the w3c approuch to role **tablist**.

- https://www.w3.org/TR/wai-aria-practices-1.1/examples/tabs/tabs-1/tabs.html
- https://www.w3.org/TR/wai-aria-practices-1.1/examples/tabs/tabs-2/tabs.html

All buttons are findings in the axe-test.

_Nested interactive controls are not announced by screen readers_

**Comment by deleonio** [comment]
We used shadow dom by buttons ... so the role tab overwrites sub element button. We fixed it ... everything is okay with axe core.

**Comment by straker** [comment]
@deleonio does that mean I can close this issue?

**Comment by straker** [comment]
Going to close this issue. Let us know if there's still an issue with the rule.
